features/attached_commands.feature

```diff
@@ -213,7 +213,7 @@ Feature: Command behaviour when attached to an UA subscription
             """
             This command must be run as root \(try using sudo\).
             """
-        When I run `ua auto-attach` with sudo
+        When I verify that running `ua auto-attach` `with sudo` exits `2`
         Then stderr matches regexp:
             """
             This machine is already attached
```

Comment:
Shouldn't this now be something like:
When I run `ua auto-attach` with sudo
Then stderr matches regexp:
"""
Skipping attach: Instance 'i-[0-9a-z]+' is already attached.
""""

Reply:
In this specific case we are not in a PRO instance, so I think the return code should be `2`

Reply:
@blackboxsw maybe it would be nice to add a new test step in the PRO tests, where we cover the case when it returns `0`. Will put that up.

Reply:
Thanks for the PRO coverage on this.